index.php

```diff
@@ -316,11 +328,11 @@ public function submit_color() {
 					print esc_attr( 'Sorry, your nonce did not verify.' );
 					exit;
 				} else {
-					$snippet_box_bg   = esc_attr( $_POST['snippet_box_bg'] );
-					$snippet_title_bg = esc_attr( $_POST['snippet_title_bg'] );
-					$border_color     = esc_attr( $_POST['snippet_border'] );
-					$title_color      = esc_attr( $_POST['snippet_title_color'] );
-					$box_color        = esc_attr( $_POST['snippet_box_color'] );
+					$snippet_box_bg   = esc_attr( sanitize_text_field( $_POST['snippet_box_bg'] ) );
```

Comment:
No need to escape and sanitize POST variable, Just sanitize will work.

Reply:
changes have done.